CONTRIBUTING.md

```diff
@@ -107,8 +108,8 @@ To help users see value in updating the SDK, we maintain a changelog file with e
 We add the heading in the first PR that's adding either a feature or fixes in the current release.
 After a release, the [changelog file will contain only the last release entries](https://github.com/getsentry/sentry-dotnet/blob/main/CHANGELOG.md).
 
-When you open a PR in such case, you need to add a heading 2 named `## Unreleased`, which is replaced during release with the version number chosen.
-Below that, you'll add the heading 3 mentioned above. For example, if you're adding a feature "Attach screenshots when capturing errors on WPF", right after a release, you'd add to the changelog:
+When you open a PR in such cases, you need to add a heading 2 named `## Unreleased`, which is replaced during release with the version number chosen.
+Below that, you'll add heading 3 mentioned above. For example, if you're adding a feature "Attach screenshots when capturing errors on WPF", right after a release, you'd add to the changelog:
 
 ```
 ## Unreleased
```

Comment:
suggestion: While at it ... let's update the example below:

We use `-` instead of `*` for markdown lists.

```diff
 ## Unreleased

 ### Features

-* Attach screenshots when capturing errors on WPF (#PR number)
+- Attach screenshots when capturing errors on WPF (#PR number)
```